src/locales/co.json

```diff
@@ -30,7 +30,19 @@
                 "courses"   : "Electivas",
                 "study-plan": "Currículos"
             }
-        }
+        },
+        "insa-access": {
+            "title": "Acceso al INSA",
```

Comment:
Can you do some research in universities to see how they handle sections like this ?

Reply:
I did and I saw that section on the oxford website :)